enclave_manager/avalon_enclave_manager/enclave_manager.py

```diff
@@ -330,6 +323,71 @@ def create_enclave_signup_data():
     return enclave_signup_data
 
 
+# -----------------------------------------------------------------
+async def wo_process(config, kv_helper, enclave_manager):
+    """
+    Process work order requests in TEE and persist result in KvStorage
+
+    @param enclave_manager - instance of the EnclaveManager class
+    @param kv_helper - instance of KvStorage
+    @param config - instance of the configuration parsed while bringing up
+                    enclave manager service
+    """
+    try:
+        sleep_interval = int(config["EnclaveManager"]["sleep_interval"])
```

Comment:
Need to integrate the change for synchronous processing via ZMQ instead of polling based on the config

Reply:
Yes, will rebase my PR.